src/sidebar/components/test/view-switcher-test.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var angular = require('angular');
+
+var util = require('../../directive/test/util');
+
+describe('selectionTabs', function () {
+  before(function () {
+    angular.module('app', [])
+      .component('selectionTabs', require('../selection-tabs'));
```

Comment:
References to `selectionTabs` and `selection-tabs.js` need replacing with `viewSwitcher` / `view-switcher.js`.